adapter/oci/provider.go

```diff
@@ -0,0 +1,14 @@
+package oci
+
+import (
+	"github.com/fnproject/cli/adapter"
+	"github.com/oracle/oci-go-sdk/functions"
+)
+
+type Provider struct {
+	FMCClient *functions.FunctionsManagementClient
```

Comment:
Very minor nit: Could this just be `FMClient`? The extra `C` seems redundant

Reply:
Fixed.